src/main/java/seedu/address/logic/parser/FindConcertCommandParser.java

```diff
@@ -20,15 +20,9 @@ public class FindConcertCommandParser implements Parser<FindConcertCommand> {
      * @throws ParseException if the user input does not conform the expected format
      */
     public FindConcertCommand parse(String args) throws ParseException {
-        String trimmedArgs = args.trim();
-        if (trimmedArgs.isEmpty()) {
-            throw new ParseException(
-                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindConcertCommand.MESSAGE_USAGE));
-        }
-
         ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_NAME);
 
-        if (argMultimap.getValue(PREFIX_NAME).isEmpty()) {
+        if (argMultimap.getValue(PREFIX_NAME).isEmpty() || !argMultimap.getPreamble().isEmpty()) {
```

Comment:
Is there a need to fix this? Cos for other commands it doesn't matter
You can do like
list  gfdbdfcg dfcggerw34t df
And it still works

Reply:
If wanna fix this, means we probably need to fix for other cases also to be consistent